src/writer.js

```diff
@@ -123,12 +127,21 @@ class Writer {
     this._out('<<');
     const keys = Object.keys(dict);
     keys.forEach((k, i) => {
+      if (k === 'Stream') {
+        return;
+      }
       this.any(k)._out(' ').any(dict[k]);
       if (i + 1 !== keys.length) {
         this._out(' ');
       }
     });
     this._out('>>\n');
+    keys.forEach((k, i) => {
+      if (k === 'Stream') {
+        this.stream(dict[k], true);
+        return;
+      }
+    });
```

Comment:
We can have this instead of forEach
`if (dict.Stream) {
   this.stream(dict[k], true);
}`